megamek/src/megamek/client/bot/princess/Princess.java

```diff
@@ -2098,6 +2105,55 @@ private void unloadTransportedInfantry(MovePath path) {
         }
     }
     
+    /**
+     * Helper function that adds an "launch" step for units that are transporting 
+     * launchable units in some kind of bay.
+     */
+    private void launchFighters(MovePath path) {
+        // if my objective is to cross the board, even though it's tempting, I won't be leaving the infantry
+        // behind. They're not that good at screening against high speed pursuit anyway.
+        if(getBehaviorSettings().shouldGoHome()) {
```

Comment:
Spacing... Please try to fix this part of your style, as it is very inaccessible